src/data.js

```diff
@@ -0,0 +1,31 @@
+import { renderInfo, setAttribute, changeBackground } from './domManipulation';
+
+const apiInfo = (location) => {
```

Comment:
Kindly use either Async and Await or Promise in getting the API info. For example; `const apiInfo = async (location) => {`

Reply:
I did use Async and Await in getting the API info. The consumeData function is the one I use and it's the only function that I export from the data.js Module. And it's an asynchronous function.